libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/RouteProgress.kt

```diff
@@ -67,6 +69,7 @@ class RouteProgress internal constructor(
     val stale: Boolean,
     val routeAlternativeId: String?,
     val currentRouteGeometryIndex: Int,
+    val nextLegWaypoint: LegWaypoint?,
```

Comment:
is the **next** leg waypoint is the destination point of the **current** route leg?

Reply:
What do you think about name `currentLegDestination`?